test/stampit-api.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+var stampit = require('../stampit'),
+  test = require('tape');
+
+// Main API
+
+test('stampit()', function (t) {
+  t.equal(typeof stampit(), 'function', 'Should produce a function.');
+
+  t.end();
+});
+
+test('stampit({})', function (t) {
+  t.ok(stampit.isStamp(stampit({})));
+
+  t.end();
+});
+
+test('incorrect stampit({ methods }) args', function (t) {
```

Comment:
Good thinking. Just to clarify, these all result in stamps with no methods, right?